src/lib/Libattr/attr_func.c

```diff
@@ -1584,3 +1584,24 @@ void free_attrl_list(struct attrl *at_list)
 
 }
 
+/**
+ * @brief	Helper function to remove a flag from an array of attributes
+ *
+ * @param[in]	pattr - pointer to the attribute list
+ * @param[in]	flag - the flag to unset
```

Comment:
There is nothing stopping you from passing in more than one flag to use this function, I'd rename this variable 'flags'